spec/state_machines/form_state_machine_spec.rb

```diff
@@ -39,15 +39,27 @@ class FakeForm < Form
       end
     end
 
-    context "when form is live_with_draft" do
```

Comment:
Have we lost the test for the `live_with_draft` -> `live` transition?

Reply:
Oops, good spot! Have readded